### PR TITLE
fix(image): add height prop and set loading=auto if no height is present

### LINKED ISSUE #194: Image component doesn't load images if height is not present
**_Bug report_**

- [x] I've searched for existing issues
- [x] I've checked documentation: https://docs.frontity.org
- [x] I've checked community forum: https://community.frontity.org

**Describe the expected behavior**

1. The `<Image>` component should render images even if no `height` is provided.
1. The `<Image>` component should not output a warning if `height` is provided.

**Describe the observed behavior**

1. The `<Image>` component needs a height to work with `loading=lazy`. If it is not provided, the image doesn't load (at all).
1. The `<Image>` component doesn't set the `height` attribute to the `` tag and a warning is sent.

**Describe the steps involved to reproduce the problem**

1. Add an `<Image>` component without height: it should not appear.
1. Add an `<Image>` component with height: the warning should appear.

**Possible solutions**

- Deactivate `loading=lazy` if height is not provided.
- Use IO if height is not provided.
- Add a default `min-height`: this won't remove the warning, but the image will appear.
- Add a default `aspectRatio`: this won't remove the warning but the image will appear. Maybe in the future with `intrinsicSizes`.

### PR DESCRIPTION
**Pull request by DAreRodz**
#### Description of what you did:

- Add a `height` prop to the `Image` component.
- Set the `loading` prop to `auto` if the`height` property is not set, avoiding Chrome to lazy-load the image.

Fixes #194.

#### My PR is a:

- 🐞 Bug fix

#### Is the PR ready to be merged?

- ✅ Yes!

### CODE REVIEW

**Comment by luisherranz** [comment]
> #### My PR is a:
> * 🐞 Bug fix
> * 🚀 New feature
> * 🔝 Improvement
> 
> #### Is the PR ready to be merged?
> * ✅ Yes!
> * 🚧 Work in progress

Isn't it clear that you should remove the non-relevant ones? And leave it like this? 😓 

> #### My PR is a:
> * 🐞 Bug fix
> 
> #### Is the PR ready to be merged?
> * ✅ Yes!

**Comment by luisherranz** [comment]
I removed the checkboxes because GitHub thinks it's progress:
Screen Shot 2019-09-18 at 19 45 52

**Comment by luisherranz** [comment]
It'd be amazing to take this opportunity to add e2e tests for the image component.

**Comment by DAreRodz** [comment]
Yeah, I'm working on that.

**Comment by luisherranz** [comment]
Let me know once this is ready.

File: e2e/packages/image/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import Image from "@frontity/components/image";
+import Package from "../types";
+
+type ImageProps = Parameters<typeof Image>["0"];
+
+const props: ImageProps = {
+  alt: "gullfoss",
+  src: "https://test.frontity.io/wp-content/uploads/2016/11/Gullfoss.jpg",
```
Comment (luisherranz) [inline e2e/packages/image/src/index.tsx]: Could you please use a local images? I don't want this test to fail if test.frontity.io is offline or if those images are removed. You can download them and add them to the `"image"` package.

**Review by luisherranz — CHANGES_REQUESTED** [review]

**Comment by luisherranz** [comment]
I love the `useInView` e2e tests! Good work 👏

**Comment by DAreRodz** [comment]
@luisherranz, can you check again this PR?

**Review by luisherranz — APPROVED** [review]
Approved!

**Pull request merged**